install_config/http_proxies.adoc

```diff
@@ -55,6 +55,18 @@ For node hosts::
 For the Docker service::
 * Registry service IP and host name
```

Comment:
For the Docker service:

* Registry service IP 
* Registry service URL  "docker-registry.default.svc.cluster.local"
* Registry route hostname (if created)